core/src/main/java/com/sequenceiq/cloudbreak/core/flow2/service/TerminationTriggerService.java

```diff
@@ -58,7 +59,7 @@ private void handleIfStackIsNotTerminated(Stack stack, boolean forced) {
             LOGGER.debug("Found termination flowlog with id [{}] and payload [{}]", flowLog.getFlowId(), flowLog.getPayload());
             handleIfFlowLogExistsForTermination(stack, forced, flowLog);
         } else {
-            LOGGER.warn("Couldn't find FlowLog with 'PRE_TERMINATION_STATE'. Triggering termination");
+            LOGGER.warn("Couldn't find FlowLog with 'INIT_STATE'. Triggering termination");
```

Comment:
it is not informative. In the logs, we cannot find easily what flow does not have `INIT_STATE`

Reply:
it's for indicating which path the code took if we are investigating an issue

Reply:
but it should be debug level